src/main/java/io/seqera/tower/cli/responses/runs/RunViewMetrics.java

```diff
@@ -92,35 +187,79 @@ private void processDataTable(List<Map<String, Object>> metricData, PrintWriter
         table.print();
     }
 
-    @Override
-    public void toString(PrintWriter out) {
-        List<String> fields = columns.stream().map(Enum::name).collect(Collectors.toList());
-        List<String> cols = new ArrayList<>();
-        cols.add("process");
-        cols.add("metric");
-        cols.addAll(fields);
+    /**
+     * Process data into a summarized or condensed table.
+     *
+     * @param metricData
+     * @param out
+     * @param cols
+     * @param formatter
+     */
+    private void processDataReducedTable(List<Map<String, Object>> metricData, PrintWriter out, List<String> cols, Map<String, Function<Float, Object>> formatter) {
+        TableList table = new TableList(out, cols.size(), cols.toArray(new String[0])).sortBy(0);
+        table.setPrefix("    ");
 
-        if (!metricsMem.isEmpty()) {
-            out.println(ansi(String.format("%n%n    @|bold  Memory Metrics|@%n    ----------------%n")));
-            processDataTable(metricsMem, out, cols);
-        }
+        metricData.forEach(processDataBlock -> {
+            processDataBlock.forEach((process, sectionDataBlock) -> {
+                List<String> cells = new ArrayList<>();
+                cells.add(process);
+                Map<String, List<String>> data = summarizeDataBlocks((Map<String, Map<String, Float>>) sectionDataBlock, formatter);
+                if (data.size() > 0) {
 
+                    // This where summarized data cells are created into a concatenated string.
+                    data.values().stream().forEach(it -> {
+                        cells.add(String.join(" / ", it));
```

Comment:
Given that we want a condensed view, I think that is better to use just a `" "` blank space to separate the different values.